README.md

```diff
@@ -1,101 +1,77 @@
-# Welcome to Viam Documentation
+# Welcome to the Viam Documentation
 
-## Dependencies
+> **Note**
+> Looking to contribute? Check out the [Contributer Guide](./CONTRIBUTING.md).
 
-* npm
-  * Install 16 (LTS) from [https://nodejs.org/en/download/](https://nodejs.org/en/download/).
+## Build the docs locally
 
-* Hugo
-  * macOS/Linux: `brew install hugo`
-  * Windows: [https://gohugo.io/getting-started/installing/](https://nodejs.org/en/download/)
+To be able to build the docs locally, you need to install the following:
 
-## Local Development
+- [`npm`](https://nodejs.org/en/download/).
+- Hugo
+  - macOS/Linux: `brew install hugo`
+  - Windows: [https://gohugo.io/getting-started/installing/](https://nodejs.org/en/download/)
 
-* Production views
-  * Standard production: `make serve-prod`
-  * Production with draft files: `make serve-prod-draft`
-  * Production with future files: `make serve-prod-future`
```

Comment:
Just a question: have we ever used future files? Maybe we should use them to help sync with eng changes?? Though I guess that would require locked-in eng change dates

Reply:
I don't think we've used them yet. We can still run the commands (they're in the `Makefile`)